extensions/amp-story-auto-ads/0.1/story-ad-page.js

```diff
@@ -48,6 +53,9 @@ const TIMEOUT_LIMIT = 10000; // 10 seconds
 /** @const {string} */
 const GLASS_PANE_CLASS = 'i-amphtml-glass-pane';
 
+/** @const {string} */
+const DESKTOP_FULLBLEED_CLASS = 'i-amphtml-story-desktop-fullbleed';
```

Comment:
Should we add another prefix to make it different from the class the AMP Story runtime uses? I'm afraid of CSS properties collisions eventually